src/components/SlideshowHero/_slideshow-hero.scss

```diff
@@ -89,6 +92,10 @@
   margin-bottom: var(--space-xl);
   transition: opacity ease var(--transition-duration);
   width: 100%;
+
+  @include mq($until: sm) {
+    margin-bottom: calc(2* var(--space-unit));
```

Comment:
```css
2* var(--space-unit)
```

is the linter catching the space before the `*` and removing it? 

imo it would read better with spaces either side of the `*`

Reply:
I just omitted the space
def more readable with it
done
🎸